ci-operator/step-registry/openshift/e2e/gcp/libvirt/conf/openshift-e2e-gcp-libvirt-conf-commands.sh

```diff
@@ -14,12 +14,18 @@ cat > "${SHARED_DIR}"/test-list << EOF
 [Conformance][Area:Networking][Feature:Router] The HAProxy router should run even if it has no access to update status [Suite:openshift/conformance/parallel/minimal]
 [Conformance][templates] templateinstance creation with invalid object reports error  should report a failure on creation [Suite:openshift/conformance/parallel/minimal]
 [Conformance][templates] templateinstance cross-namespace test should create and delete objects across namespaces [Suite:openshift/conformance/parallel/minimal]
+[sig-devex][Feature:Templates] templateinstance cross-namespace test should create and delete objects across namespaces [apigroup:user.openshift.io][apigroup:template.openshift.io] [Suite:openshift/conformance/parallel]
```

Comment:
All the duplicated tests in this file got renamed through https://github.com/openshift/origin/pull/24683 more than 2 years ago (Mar 13, 2020). CC @praveenkumar @sallyom